src/server/serve-static.ts

```diff
@@ -11,7 +11,7 @@ import { ComputeJsNextRequest, ComputeJsNextResponse } from "./base-http/compute
 
 /**
  * Serves the contents of a file at a path.
- * (A reimplementation for Compute@Edge of function in Next.js of the same name,
+ * (A reimplementation for Compute of function in Next.js of the same name,
```

Comment:
Is this "of" meant to be here?

Reply:
yes, this is correct.  a function with this name exists in Next.js, and this is a reimplementation for Compute of that function.